yarn.lock

```diff
@@ -9274,6 +9358,16 @@ immer@1.10.0:
   resolved "https://registry.yarnpkg.com/immer/-/immer-1.10.0.tgz#bad67605ba9c810275d91e1c2a47d4582e98286d"
   integrity sha512-O3sR1/opvCDGLEVcvrGTMtLac8GJ5IwZC4puPrLuRj3l7ICKvkmA0vGuU9OW8mV9WIBRnaxp5GJh9IEAaNOoYg==
 
+immer@^8.0.0:
```

Comment:
Where is this used @kelvin-muchiri ?

Reply:
@moshthepitt `@opensrp/pkg-config` uses `@reduxjs/toolkit` which depends on `immer`